src/logic.js

```diff
@@ -216,15 +216,18 @@ define([
                     pathString = isHashtag ? path.toHashtag() : path.pathWithoutPredicates(),
                     pathWithoutRoot = isHashtag ? '' : pathString.substring(1 + pathString.indexOf('/', 1)),
                     refMug = form.getMugByPath(pathString),
-                    xpath = path.toHashtag();
+                    xpath = path.toHashtag(),
+                    knownHashtag = pathString.startsWith('#case') && _.contains(form.validHashtags(), xpath);
```

Comment:
It's not clear to me why we are treating `#case` hashtags specially here. It seems that we may have other external hashtag references in the future that may need to hit this same code path. Would it make sense to do `pathString.startsWith('#')`?